apps/dms-material/src/app/shared/components/base-table/base-table.component.scss

```
@@ -78,18 +98,62 @@ mat-progress-bar {
-  border-spacing: 0;
+// Header region — NOT position:sticky. Always above CDK viewport in document flow.
+.dms-table-header {
+  flex-shrink: 0;
 }
 
-tr.mat-mdc-row {
+.dms-header-row {
+  display: flex;
+  flex-direction: row;
+  background-color: var(--dms-surface);
+  // box-shadow replaces the collapsed border line that sticky <th> borders provided:
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)